### ISSUE TITLE
[Bug] Floorp not compatible with allegro.pl website. I'm blocked there, only by using Floorp.

### ISSUE THREAD

**Issue by mahcin** [Floorp-Projects/Floorp#721]
### Attention!

- [X] I understand that Floorp is NOT supported on Windows 7/8.1 or lower.

### Describe the bug

I have switched OFF all security, re-installed Floorp, used Floorp default settings, used Yokoffing, done everything what's possible and here is no way to use Floorp for allegro.pl service.

### To Reproduce

In Firefox it does works fine, in Floorp it doesn't --> I can open 1st page which is allegro.pl then I can click to open something on allegro.pl so 2nd page, then click to open something so 3rd page on allegro.pl then click on something and 4th page won't open. In this stage allegro.pl doesn't work for me in any other web browser so it sounds like I'm blocked by server level. If I reset router now to refresh random IP, it does work again until I hit 4th page. It doesn't happening in Firefox or Opera in same PC under same session, so certainly something in Floorp. I have added allegro.pl to all exceptions but still no joy. 

### Expected behavior

Any chance that you could have a look please at this issue?
Thanks in advance
Martin

### Screenshots

![floorp](https://github.com/Floorp-Projects/Floorp/assets/155994915/f28ce7fe-5620-430f-b16d-620798027bef)

### OS Info

Windows 10 22H2

### Floorp Version

Version 11.8.1 (64-bit) | Firefox: 115.7.0

**Comment by surapunoyousei** [comment]
![image](https://github.com/Floorp-Projects/Floorp/assets/73892113/d9841083-be0a-41a2-bb50-6dd731bec00f)

**Comment by mahcin** [comment]
yes, it does work for 1st page, but if you click on several pages, (3+) then it looks like server will block you out :-(

**Comment by surapunoyousei** [comment]
Maybe Floorp blocks by the site owner. Please contact site owner. The block is not able to resoved by Floorp.

**Comment by mahcin** [comment]
That's more likely correct. Allegro.pl is blocking Floorp. I don't know why :-(
Please look at the photo, as a proof.

![allegro_block](https://github.com/Floorp-Projects/Floorp/assets/155994915/d6ed1dc0-198b-4010-9343-40ec3083a6db)

**Comment by mahcin** [comment]
Sorry for re-opening, I just wanted to add in case if someone else will be reading it:
I'm still using Floorp, now fully daily and here is no way that I would switch to other browser. Above just shows how Floorp is good with prevention of DATA as it looks like above www wants more details than they should have :-D
I can use for above www different temporary browser, but for daily use that's definitely Floorp is number 1.

Well done **surapunoyousei**
Thank you so much for creating such good Floorp browser.

Now you can close this inquiry, thanks :-)

**Comment by TommyV6** [comment]
I got same problem. But in my case floorp stops loading 3-4 page but there is no error. After click link 3-4 time it some times works. On same machine firefox works flawless.

**Comment by mahcin** [comment]
**TommyV6**
I have spent many many hours researching and testing it. I have contacted Floorp developer regarding this issue.
I can make you assure that's not Floorp problem or fault at all.
This problem is on Allegro.pl side.
So only place where we can look for a solution is Allegro, who won't do anything as it looks like they like to take our DATA.
That's why I do love Floorp even more.

**Comment by TommyV6** [comment]
I found another site: https://chat.openai.com/ chatbox disable after one question answered.

**Comment by creeper-0910** [comment]
I cannot reproduce that problem.

**Comment by regularenthropy** [comment]
Cannot confirm either. Maybe next ESR will adress this.

**Comment by kgorol** [comment]
Allegro.pl works for me, but is VERY sluggish. Like I need to click on some link multiple times to load it, some parts of website load partially or not at all. And worst of it is after a couple of clicks, it asks me to solve a Captcha.

When next ESR is planned?

**Comment by creeper-0910** [comment]
After the next Firefox ESR is released.

**Comment by kgorol** [comment]
> After the next Firefox ESR is released.

I can confirm that on newest 1.17.3 Floorp based on ESR 128 allegro.pl still misbehaves.

**Comment by mahcin** [comment]
I have changed title as this is problem with naughty Allegro.pl who after my research wanted to take more than suppose to take - not Floorp. Nothing wrong with Floorp at all. I just use separate browser for Allegro and other "doggy" sites.
